### PR TITLE
Updated MessageKit.podspec to use resource_bundle instead of resources

### LINKED ISSUE #1459: Convert s.resource to use s.resource_bundles in pod spec file
**Describe the bug**
` s.ios.resources = ['Sources/Assets.xcassets']` triggers CocoaPods/CocoaPods#8431 which can cause apps to pull in excessive amounts of resources.

**Expected behavior**
should use  `resource_bundles` in pod spec file

### PR DESCRIPTION
**Pull request by martinpucik**
What does this implement/fix? Explain your changes.
---------------------------------------------------
- Changed `resources` to `resource_bundle` in podfile spec
- Looks like this is the preferred way to include resources in a Pod
- Reported by @wzio, would love a look at this :) 

Does this close any currently open issues?
------------------------------------------
Closes #1459 

Any relevant logs, error output, etc?
-------------------------------------
🙅‍♂️

Any other comments?
-------------------
🙅‍♂️

Where has this been tested?
---------------------------
**Devices/Simulators:** …

**iOS Version:** …

**Swift Version:** …

**MessageKit Version:** …

### CODE REVIEW

**Comment by wzio** [comment]
if change to resource_bundle 
you also have to change code in 
```
internal extension Bundle {
    #if IS_SPM
    static var messageKitAssetBundle: Bundle = Bundle.module
    #else
    static var messageKitAssetBundle: Bundle = Bundle(for: MessagesViewController.self)
    #endif
}
```
To get right bundle

**Review by Kaspik — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -15,6 +15,7 @@ The changelog for `MessageKit`. Also see the [releases](https://github.com/Messa
 ### Changed
 
 - **Breaking Change** Dropped support for iOS 11, added support for Swift 5.3 SPM and XCode 12 [#1464](https://github.com/MessageKit/MessageKit/pull/1464) by [@kaspik](https://github.com/kaspik)
+- Changed `resources` to `resource_bundle` in MessageKit.podspec [#1460](https://github.com/MessageKit/MessageKit/pull/1460) by [@martinpucik](https://github.com/martinpucik)
```
Comment (Kaspik) [inline CHANGELOG.md]: Wrong place :))

File: CHANGELOG.md [hunk 2]
```diff
@@ -14,6 +14,10 @@ The changelog for `MessageKit`. Also see the [releases](https://github.com/Messa
 
 - Fixes an issue where the MessagesViewController keyboard observers were not cleaned up when MessagesViewController was no longer visible on screen [#1476](https://github.com/MessageKit/MessageKit/pull/1476) by [@kinoroy](https://github.com/kinoroy)
 
+### Changed
+
+- Changed `resources` to `resource_bundle` in MessageKit.podspec [#1460](https://github.com/MessageKit/MessageKit/pull/1460) by [@martinpucik](https://github.com/martinpucik)
```
Comment (Kaspik) [inline CHANGELOG.md]: Wrong place again now. 😂 Sorry Martin!

Comment (martinpucik) [reply CHANGELOG.md]: 😹

**Review by Kaspik — APPROVED** [review]

**Comment by chunxige** [comment]
please update the 3.3.0 version to support ios11

**Pull request merged**